base/mpfr.jl

```diff
@@ -176,9 +176,9 @@ struct BigFloatData{Limb}
 end
 
 # BigFloat interface
-@inline function Base.getproperty(x::BigFloat, s::Symbol)
+@inline function etproperty(x::BigFloat, s::Symbol)
```

Comment:
```suggestion
@inline function getproperty(x::BigFloat, s::Symbol)
```
But actually I think you do need the qualifications for the method extensions, since this is in a module that hasn't `import`ed `getproperty`.